markdown/3.x/en/faq.md

```diff
@@ -100,6 +100,26 @@ If this doesn't seem to solve the issue, please contact us for a custom solution
 
 ---
 
+#### Question:
+Can i contribute?
+
+#### Answer:
+Yes. Although the Web Components codebase is closed, you can still discuss and propose at [GitHub issues](https://github.com/FACT-Finder-Web-Components/ff-web-components/issues).
```

Comment:
[off-topic]
I just imagined someone proposing through a GitHub issue and the other half closing it as "won't do" 🤣

Reply:
What a rejection! 😆